releases-istio/secrets/docker-cfg-secret.yaml

```diff
@@ -1,15 +1,8 @@
 apiVersion: v1
-stringData:
-  config.json: |-
-    {
-      "auths": {
-        "docker-delivery.DOMAIN:443": {
-          "auth": "dXNlci1hZG1pbjpMOFRjbnJ3TXl0Q0ZwQUZl"
-        }
-      }
-    }
+data:
+  .dockercfg: <concealed>
```

Comment:
Should we provide here placeholder, so we can replace?

Reply:
Not yet, we will have to create a config for this and then convert that config to secret